src/index.js

```diff
@@ -0,0 +1,41 @@
+// import './style.css';
+import './style.css';
+
+// creating tasks
+const tasks = [{
+  description: 'Gym',
+  completed: true,
+  index: 0,
+}, {
+  description: 'Code',
```

Comment:
- [ ] Please, consider removing the comments to make the code clear - your code is understandable and talks about what happening 👍